pyiron_base/database/performance.py

```diff
@@ -79,9 +78,8 @@ class DatabaseStatistics:
     """
 
     def __init__(self):
-        s = Settings()
-        connection_string = s._configuration['sql_connection_string']
-        self._job_table = s._configuration['sql_view_table_name']
+        connection_string = state.settings.configuration['sql_connection_string']
```

Comment:
That is an *extremely* good catch, thank you Muhammad! Not only did we separate them, but the on stored in settings now stays as the raw input the user gave, and the one in the database is fancified, so this would have actually broken. I did string searching for all the keys that `database` is stealing from `settings.configuration` and now reference the database directly (although the `sql_connnection_string` is the only one that is not identical)